.travis.yml

```diff
@@ -44,3 +58,6 @@ jobs:
         on:
           tags: true
           repo: jazzband/pip-tools
+
+  allow_failures:
+    - env: PIP=master
```

Comment:
I think this makes a nicer contributor experience, as the Github status wouldn't be "failed".

Let me know if you'd rather not have that, I'll take it out.

Reply:
I think this is a good addition, thank you.
If it's possible to do the same on Appveyor, that would be great